From 0da95b668d943c84cc86cf970f1e13649e2fcbbc Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Wed, 23 Nov 2022 11:51:42 +0100
Subject: [PATCH 087/529] ARM: configs: enable protection consumer in STM32MP1
 addons fragment

Enable Regulator protection consumer in local Linux config fragment.

Signed-off-by: Pascal Paillet <p.paillet@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/204696
Reviewed-by: Alexandre TORGUE <alexandre.torgue@foss.st.com>
Change-Id: I438e88bf66d563d8285f78af9803bb5764de6139
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/275803
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Domain-Review: Alexandre TORGUE <alexandre.torgue@foss.st.com>
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -35,6 +35,7 @@ CONFIG_PREEMPT=y
 CONFIG_PREEMPT_COUNT=y
 CONFIG_PROFILING=y
 CONFIG_REGULATOR_ARM_SCMI=y
+CONFIG_REGULATOR_PROTECTION_CONSUMER=y
 CONFIG_SCHED_MC=y
 CONFIG_SENSORS_IIO_HWMON=y
 CONFIG_SERIAL_NONSTANDARD=y
